### ISSUE TITLE
Enable tracing from Step Functions to flow to Open Telemetry Collector

### ISSUE THREAD

**Issue by rberger** [aws-observability/aws-otel-community#588]
As far as I can tell, there does not seem to be a way to have Step Functions participate in an Open Telemetry Trace. We are adopting Step Functions very heavily now and also adopting Open Telemetry. Had expected that there would be a way for Step Functions to send trace data to Open Telemetry, but it seems it can only send to X-Ray.

Is there a way to have that Step Function trace info be sent to Open Telemetry? If not, is there maybe a way to have the Step Function X-Ray traces sent to Open Telemetry?

**Comment by pandarun** [comment]
Hi, probably similar issue https://github.com/aws-observability/aws-otel-collector/issues/1845#issuecomment-1448592271.

Did you manage to find a workaround?

**Comment by rberger** [comment]
We would like to try again to use Open Telemetry but now Step Functions are core to our system and would need a solution to have Step Function actions flow into Open Telemetry traces.  Has there been any effort to include mechanisms so Step Functions are part of OTel traces?

**Comment by kollmats** [comment]
Have the same problem. @rberger have you found a way forward yet?

**Comment by rberger** [comment]
> Have the same problem. @rberger have you found a way forward yet?

Nope, we still can't really use Open Telemetry due to this as Step Functions are core to our entire infrastructure. We are kind of using X-Ray

**Comment by nine5two7** [comment]
@rberger Just a quick note in case you're still looking for it—Datadog now offers a solution for tracing Step Functions, including support for distributed tracing with proper instrumentation. You can find more details on [this page](https://docs.datadoghq.com/serverless/step_functions/):
